README.md

```diff
@@ -293,14 +313,14 @@ generate from your probes. It writes those incoming reports as JSON
 lines to a file. Start it like so:
 
 ```
-$ cd collectors/modality-probe-udp-collector
+$ cd ../collectors/modality-probe-udp-collector
 $ cargo install --path .
-$ cd ../../
+$ cd ../../modality-probe-capi
 $ modality-probe-udp-collector
 Using the configuration:
     addr: 0.0.0.0:2718
     session id: 0
-    output file: /home/me/src/modality-probe/session_0_log_entries.jsonl
+    output file: /home/me/src/modality-probe/modality-probe-capi/session_0_log_entries.jsonl
```

Comment:
The `modality-probe-capi` directory is where the source for the C API lives, we shouldn't be putting stuff in there. Let's leave this file at the root.

Reply:
:+1: Sure, this is another inheritance from `modality-probe-capi` I assume the original intent was to account for a hypothetical user doing both the C and Rust examples; not an issue for now. Updating to:

`output file: /home/me/src/modality-probe/session_0_log_entries.jsonl`